src/app/child-dev-project/notes/notes-related-to-entity/notes-related-to-entity.component.ts

```diff
@@ -90,12 +90,14 @@ export class NotesRelatedToEntityComponent
         }
       }
 
-      for (const e of [
-        this.entity.getId(),
-        ...this.getIndirectlyRelatedEntityIds(this.entity),
-      ]) {
-        if (!newNote.relatedEntities.includes(e)) {
-          newNote.relatedEntities.push(e);
+      if (this.entity.getType() !== "Child") {
+        for (const e of [
```

Comment:
This was adding all entity types, including Child, to the `relatedEntities` array. To prevent that, I'm now excluding the Child entity, and it is working fine.